### ISSUE TITLE
Classes with static methods only 

### ISSUE THREAD

**Issue by artemp** [mapbox/geojson-vt-cpp#37]
There are few classes declarations with static members only e.g
```
// This class has only static functions; disallow creating instances of it.
    Clip() = delete;

```
https://github.com/mapbox/geojson-vt-cpp/blob/master/include/mapbox/geojsonvt/clip.hpp
is this for some reason ^ ? 

In c++ the same behaviour better achieved by simply making a standalone methods and adding enclosing namespace - `namespace Clip {} ` for example.

/cc @kkaefer @mourner @springmeyer @jfirebaugh

**Comment by mourner** [comment]
Yeah, I think we can convert those to simple namespaced functions. I see that they were classes in the first commit.

**Comment by kkaefer** [comment]
Yeah, we should convert them to standalone functions.

**Comment by mourner** [comment]
Addressed by #45.
